### PR TITLE
PeoplePicker: Improved keyboard support

### PR DESCRIPTION
**Pull request by kristoferbrown**
#### Pull request checklist

- [x ] Include a change request file using `$ npm run change`

#### Description of changes

Allows the 'Load all Results' button to get keyboard focus.
recording

Prevents keyboard focus from skipping the first item in the suggested contacts list.
select-first-item

Also, decreases font size of Loading and Load all Results elements.

#### Focus areas to test
The people picker suggestion list.

### CODE REVIEW

**Comment by betrue-final-final** [comment]
The second issue is fixed, testing with Narrator + Edge, but I can't get the "Load all results" footer to appear.

**Comment by betrue-final-final** [comment]
Alright you can also get to "Load all results" but you can't get to "Top 10 results" after you tap "Load all results"
image

**Comment by betrue-final-final** [comment]
Ok, I talked to to Kris, and Top 10 results is just a label. Please make that label color instead of link color "neutralPrimary"

**Comment by kristoferbrown** [comment]
@betrue-final-final, I removed the Top 10 element since it wasn't really represented in the XD file and was causing this ambiguity.

File: packages/office-ui-fabric-react/src/components/Callout/Callout.scss [hunk 1]
```diff
@@ -25,6 +25,7 @@ $Callout-smallbeak-slant-width: 16px;
   @include ms-drop-shadow(0,0, 5px, 0px);
   position: absolute;
   box-sizing: border-box;
+  border: 1px solid $ms-color-neutralLight;
```
Comment (benkaiser) [inline packages/office-ui-fabric-react/src/components/Callout/Callout.scss]: As much as I want this change to happen, this isn't the right PR for it and I'd recommend splitting it out into it's own PR in an effort to keep PRs Single Responsibility.

Comment (kristoferbrown) [inline packages/office-ui-fabric-react/src/components/Callout/Callout.scss]: The border was originally removed in an update to the people picker's callout, so I didn't think it was a big deal to add it back in another update to the people picker's callout. You're right though, I'll pull it out here and start a new PR.

Comment (kristoferbrown) [inline packages/office-ui-fabric-react/src/components/Callout/Callout.scss]: @benkaiser https://github.com/OfficeDev/office-ui-fabric-react/pull/2379

**Pull request merged**